modules/sr/robot/utils.py

```diff
@@ -1,2 +1,9 @@
-def map_to_range(old_min, old_max, new_min, new_max, value):
+def map_to_range(
+    old_min: float,
```

Comment:
Is there a need to constrain this to operating on `float` over also `int`?

Reply:
Due to what this is doing the output will always be `float`, so there's nothing to be gained from generics. Since you can always pass an `int` to something expecting a `float` this is simplest to define.